packages/api/src/services/workbench-language.service.ts

```diff
@@ -0,0 +1,26 @@
+import {WorkbenchEventService} from './workbench-event.service';
+import {WorkbenchServiceProvider} from '../workbench-service.provider';
+import {LanguageChangeEvent} from '../models/events/language/language-chage.event';
+import {WorkbenchService} from './workbenchService';
+import {WorkbenchGlobalContextService} from './workbench-global-context.service';
+import {Subject} from '@reactivex/rxjs/dist/package';
+
+export class WorkbenchLanguageService implements WorkbenchService {
+    private readonly eventService: WorkbenchEventService;
+    private readonly globalContextService: WorkbenchGlobalContextService;
+
+    static readonly DEFAULT_LANGUAGE = 'en';
```

Comment:
There is something odd in these services the global context one and this. At first they seem a bit overcomplicated and second, it seems that there is some mashup in their purposes.

Reply:
The global one is removed.